src/azure-cli/azure/cli/command_modules/botservice/tests/latest/test_bot_commands.py

```diff
@@ -481,23 +481,23 @@ def test_botservice_create_should_raise_error_for_invalid_app_id_args(self, reso
             self.cmd('az bot create -g {rg} -n {botname} --appid {numbers_id} --app-type MultiTenant')
             raise AssertionError()
         except CLIError as cli_error:
-            assert cli_error.__str__() == expected_error
+            assert (str(cli_error)) == expected_error
```

Comment:
There are nnnecessary parentheses.

Reported to https://github.com/astral-sh/ruff/issues/14597